### PR TITLE
fix(layout): inline & block gutters

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

**Review by rinaok — CHANGES_REQUESTED** [review]
Didn't understand the example. Maybe it should be two examples? One with `inline` sizes and one with `block` sizes?

File: libs/components/src/lib/layout/README.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ Using layout, elements can be arranged easily in a variety of sizes and spaces t
 
 Use the `gutters` attribute to set the initial preferred `margin`.
 
-- Type: `'small'` | `'medium'` | `'large'`
+- Type: `'small'` | `'small-inline'` | `'small-block'` | `'medium'` | `'medium-inline'` | `'medium-block'` | `'large'` | `'large-inline'` | `'large-block'` 
+
 - Default: `undefined`
```
Comment (yinonov) [inline libs/components/src/lib/layout/README.md:42]: Can this be define as none in docs?

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/layout/README.md (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/layout/README.md]: done

**Comment by rachelbt** [comment]
> Didn't understand the example. Maybe it should be two examples? One with `inline` sizes and one with `block` sizes?

@yinonov @rinaok 
I added dividers, I think now its better, no?

**Comment by rinaok** [comment]
> > Didn't understand the example. Maybe it should be two examples? One with `inline` sizes and one with `block` sizes?
> 
> @yinonov @rinaok I added dividers, I think now its better, no?

@rachelbt I still think that it would be better with two different examples

**Comment by rinaok** [comment]
> > > Didn't understand the example. Maybe it should be two examples? One with `inline` sizes and one with `block` sizes?
> > 
> > 
> > @yinonov @rinaok I added dividers, I think now its better, no?
> 
> @rachelbt Looks better, but I still think that it would be better with two different examples

@rachelbt Added a change, what do you think? I think that it's more clear now

**Comment by rachelbt** [comment]
> > > > Didn't understand the example. Maybe it should be two examples? One with `inline` sizes and one with `block` sizes?
> > > 
> > > 
> > > @yinonov @rinaok I added dividers, I think now its better, no?
> > 
> > 
> > @rachelbt Looks better, but I still think that it would be better with two different examples
> 
> @rachelbt Added a change, what do you think? I think that it's more clear now

its more clear but it makes the preview code very big. 
@yinonov what do you say?

**Comment by rinaok** [comment]
> > > > > Didn't understand the example. Maybe it should be two examples? One with `inline` sizes and one with `block` sizes?
> > > > 
> > > > 
> > > > @yinonov @rinaok I added dividers, I think now its better, no?
> > > 
> > > 
> > > @rachelbt Looks better, but I still think that it would be better with two different examples
> > 
> > 
> > @rachelbt Added a change, what do you think? I think that it's more clear now
> 
> its more clear but it makes the preview code very big. @yinonov what do you say?

@rachelbt Maybe you can move the gutters examples below the Column Basis example and then it will look better?

**Review by yinonov — APPROVED** [review]

**Review by rinaok — DISMISSED** [review]

**Review by rinaok — APPROVED** [review]

**Pull request merged**